Unreal/CarlaUE4/Plugins/Carla/Source/Carla/Actor/ActorBlueprintFunctionLibrary.h

```diff
@@ -108,6 +112,34 @@ class UActorBlueprintFunctionLibrary : public UBlueprintFunctionLibrary
       bool &Success,
       FActorDefinition &Definition);
 
+  static FActorDefinition MakeV2XDefinition();
+
+  UFUNCTION(Category = "Carla Actor", BlueprintCallable)
+  static void MakeV2XDefinition(
+      bool &Success,
+      FActorDefinition &Definition);
+
+  static FActorDefinition MakeCustomV2XDefinition();
+
+  UFUNCTION(Category = "Carla Actor", BlueprintCallable)
+  static void MakeCustomV2XDefinition(
+      bool &Success,
+      FActorDefinition &Definition);
+
+  static FActorDefinition MakeV2XDefinition();
+
+  UFUNCTION(Category = "Carla Actor", BlueprintCallable)
+  static void MakeV2XDefinition(
```

Comment:
Some methods like MakeV2XDefinition in this file are defined twice. So it gives compile errors.